### PR TITLE
feat: add SetFee transaction support

### LINKED ISSUE #28: Add `Simple` view for SetFee pseudotransactions
https://livenet.xrpl.org/transactions/5922A0BA30621C60B2B6DDBC3FF6B5BB509EB3685C4C3D56696A9FE4FE6D48A3

### PR DESCRIPTION
**Pull request by ckniffen**
## High Level Overview of Change

Adds Simple and Description views for SetFee and includes support for the XRPFees amendment.

Closes #28

### Type of Change

- [x] New feature (non-breaking change which adds functionality)

## Before
Screenshot 2023-02-28 at 3 42 31 PM

Screenshot 2023-02-28 at 3 42 37 PM

## After
Screenshot 2023-02-28 at 3 43 26 PM

Screenshot 2023-02-28 at 3 43 55 PM

### CODE REVIEW

File: src/containers/shared/components/Transaction/SetFee/Simple.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { SetFeeInstructions } from './types'
+
+export const Simple = ({
+  data,
+}: TransactionSimpleProps<SetFeeInstructions>) => {
+  const { fee, base, increment } = data.instructions
+
+  return (
+    <>
+      <SimpleRow label="Base Fee" data-test="base-fee">
```
Comment (mvadari) [inline src/containers/shared/components/Transaction/SetFee/Simple.tsx:13]: These labels should use translations

Comment (mvadari) [reply src/containers/shared/components/Transaction/SetFee/Simple.tsx:13]: Reopening this

**Review by mvadari — COMMENTED** [review]
This should probably also include a TableDetail page

**Comment by ckniffen** [comment]
> This should probably also include a TableDetail page

This is such a rare transaction I felt it wasn't necessary.

File: src/containers/shared/components/Transaction/SetFee/Simple.tsx (hunk 1 above)
Comment (ckniffen) [reply src/containers/shared/components/Transaction/SetFee/Simple.tsx:13]: Woops

**Comment by ckniffen** [comment]
> > This should probably also include a TableDetail page
> 
> This is such a rare transaction I felt it wasn't necessary.

Additionally since this is a pseudo transaction it will only ever appear on the ledger page.

**Review by mvadari — APPROVED** [review]

**Review by justinr1234 — APPROVED** [review]

**Pull request merged**